content/cucumber/configuration.md

```diff
@@ -1,11 +1,11 @@
 ---
-menu:
-- reference
 source: https://github.com/cucumber/cucumber/wiki/cucumber.yml/
-title: cucumber.yml
+source: https://github.com/cucumber/cucumber/wiki/Environment-Variables/
+title: Additional Configuration
```

Comment:
Cucumber Configuration makes more sense to me than Additional. Additional to what? :-p

Reply:
Good point. ;) Still need to figure out where this should go btw!